src/Controls/src/Core/Application.cs

```diff
@@ -437,5 +437,7 @@ protected internal virtual void CleanUp()
 
 			NavigationProxy = null;
 		}
+
+		IReadOnlyList<object> IVisualTreeHelper.GetVisualChildren() => new List<object> { MainPage }.AsReadOnly();
```

Comment:
Should this be the Windows instead? MainPage is more of a helper method and will always create a window..

Reply:
the purpose of this PR is to help the XET team deliver their features. It should be whatever they need 🤷‍♂️